src/lib.rs

```diff
@@ -336,7 +333,7 @@ where
     }
 
     /// Gets the given key's corresponding entry in the map for in-place manipulation.
-    pub fn entry(&mut self, key: Key) -> Entry<Key, Value> {
+    pub fn entry(&mut self, key: Key) -> Entry<'_, Key, Value> {
```

Comment:
Not exactly a review, but just for curiosity, is `'_` necessary ? I have not used these but only read about them and the idea seems to be to simplify verbosity by not forcing to mention lifetimes in function signatures before using them. All unmentioned lifetimes are assumed to be bound to the function signature. So is it OK to not mention these or does that lead to compilation OR linter errors ?

Reply:
No compiler errors, it is a lint issue only with cargo fix using edition-idioms. So no biggies, they can be inferred by the compiler and are, but for readers of the code it is supposed to make the lifetimes more apparent AFAIK.